op/clip/shapes.go

```diff
@@ -91,12 +113,9 @@ func (rr RRect) path(ops *op.Ops) PathSpec {
 	return p.End()
 }
 
-// Add the rectangle clip.
-func (rr RRect) Add(ops *op.Ops) {
-	rr.Op(ops).Add(ops)
-}
-
 // Border represents a rectangular border.
+//
+// Deprecated: use clip.Stroke{Path: clip.UniformRRect(r, rr).Path(ops), Style: clip.StrokeStyle{Width: width}}.
```

Comment:
Please remove Border altogether. It is easy to replace, perhaps even with a `go fmt -r`.

Reply:
I'm not sure about the `go fmt -r` part since it doesn't seem to handle optional fields.